collections/_posts/2022-10-11-ProbabilisticUNet.md

```diff
@@ -0,0 +1,109 @@
+
+# Highlights
+
+* The objective of this paper is to learn a distribution of segmentations from an input.
+* The main innovation concerns the design of a generative segmentation model based on the combination of a U-Net with a conditional VAE.
+* The proposed framework is also capable of learning hypotheses that have a low probability and predicting them with the corresponding frequency.
+
+
+# Method
+
+## Architecture
+
+* The architecture is based on the ***conditional VAE*** whose details are provided in the following [tutorial](https://creatis-myriad.github.io/tutorials/2022-09-12-tutorial-cvae.html).
+* The innovation comes from the use of a U-Net architecture to model the distribution $$p(y \vert x,z)$$.
+* The latent vector $$z$$ is first passed to a decoder to produce a $$N$$-channel feature map with the same spatial dimensions as the segmentation map. This feature map is then concatenated with the last activation map of a U-Net before being convolved by a last layer to produce the final segmentation map with the desired number of classes.
+
+The image below provides an overview of the architecture deployed during training. The distributions $$p(z \vert x,y)$$, $$p(z \vert x)$$ and $$p(y \vert x,z)$$ displayed in blue are modeled by three distinct neural networks.
+
+![](/collections/images/probabilistic_unet/proba_unet_training.jpg)
+
+
+The following image illustrates the use of the architecture during inference.
+
+![](/collections/images/probabilistic_unet/proba_unet_inference.jpg)
+
+&nbsp;
+
+## Implementation details
+
+* The dimensions of the latent space has been experimentally fixed at $$M=6$$ and is kept fixed in all experiments.
+* The difference between the predicted segmentation and the ground truth is optimized using a Cross Entropy loss.
+* The training is done from scratch with randomly initialized weights 
+* When drawing $$m$$ samples for the same input image, the output of the prior network and the feature activations of the U-Net can be reused without the need for recomputation, making the overall strategy computationally efficient.
+
+## Performance measures
+
+* The performance of the methods was evaluated by comparing distributions of segmentations.
+* This was done through the computation of ***generalized energy distance*** whose expression is given below:
+
+$$D^2_{GED}(P_{gt},P_{out}) = 2 \, \mathbb{E} \left[d\left(S,Y\right) \right] - \mathbb{E} \left[d\left(S,S'\right) \right] - \mathbb{E} \left[d\left(Y,Y'\right) \right]$$
+
+* $$d$$ is a distance measure, $$Y$$ and $$Y'$$ are independent samples from the ground truth distribution $$P_{gt}$$, $$S$$ and $$S'$$ are independent samples from the predicted distribution $$P_{out}$$. 
```

Comment:
ces variables ne correspondent pas au 2 figures précédentes.

Reply:
Hi, Thanks :) I just have updated the text accordingly